### ISSUE TITLE
isDeviceRooted not working correctly

### ISSUE THREAD

**Issue by danielehrhardt** [kolbasa/cordova-plugin-apkupdater#29]
![image](https://user-images.githubusercontent.com/19570043/128429676-82d6b29e-149a-4bb5-9d1d-2550c544c89c.png)

When i call isDeviceRooted() on some devices i get true but when i try to install then with root i am getting a catch with the error that the Device is not rooted. I can say that the Firmware is singed with test keys. I am not sure if this helps. 

Maybe we can find a different root detection.

**Comment by kolbasa** [comment]
Hmm. I was afraid of something like that. I wanted to have a minimal solution to test the root status. I guess this is not enough. The root methods are often very different.

I'll have to include something like that: https://github.com/scottyab/rootbeer

Give me 1 or 2 days, I'll test it out.

**Comment by danielehrhardt** [comment]
> Hmm. I was afraid of something like that. I wanted to have a minimal solution to test the root status. I guess this is not enough. The root methods are often very different.
> 
> I'll have to include something like that: https://github.com/scottyab/rootbeer
> 
> Give me 1 or 2 days, I'll test it out.

If you have something to test send it over.

**Comment by kolbasa** [comment]
`isDeviceRooted()` now uses the `rootbeer` library. 

The change is now on the branch: `root-tools`:
```
cordova plugin add https://github.com/kolbasa/cordova-plugin-apkupdater#root-tools
```

**Comment by danielehrhardt** [comment]
> `isDeviceRooted()` now uses the `rootbeer` library.
> 
> ~The change is now on the branch: `root-tools`:~
> ~`cordova plugin add https://github.com/kolbasa/cordova-plugin-apkupdater#root-tools`~
> 
> Change is published with version: `2.5.0`

It does work now correctly.

**Comment by kolbasa** [comment]
Glad to hear that. Let me know if you find anything else. 👍
